### DISCUSSION TITLE
Can’t log in my computer

### DISCUSSION THREAD

**Discussion by sedd00** [elementary/triage#666] [Q&A]
Yesterday i changed the keyboard of my computer and today i wanted to use it but i couldn't get in since it tells me wrong password even tho it is the right one and idk how to get in the computer now, can someone help?

**Comment by jeremypw** [comment]
I have had this problem before - it is annoying you cannot change the keyboard layout from the greeter screen (afaik).  If you know what keyboard layout you started with (to set the password) and which one you changed to you may be able to work out what you need to type on the new layout to generate the password.  In my case it was a simple case of swapping one character since the keyboards involved were US English and UK English.   If you press `<Ctrl><Alt>F5` e.g. at the login screen you get a terminal where you can try typing your password and can see what you are typing.   If you can get online on another machine you can lookup the relevant layouts otherwise it is a matter of trial and error.  Once in, go into Settings/User Accounts and reset your password using the current keyboard.  Also go into Settings/Keyboard and remove any layouts you do not want.

**Reply by sedd00** [reply]
In my case the problem is that the password it only numbers and m sure that my password is right but still cant log in

**Reply by jeremypw** [reply]
Still worth check that what you are typing is what you think you are typing.  Check that the numlock and capslock settings are as expected.

**Reply by jeremypw** [reply]
You can try logging in at the terminal to see what error message is given.  If it is "incorrect password" then the username/password combination given does not accord with those stored by the computer.  If it is not an input issue then I guess the stored credentials may be corrupted but that is unlikely.

**Reply by jeremypw** [reply]
Did you change your physical keyboard or just the layout setting?

**Reply by sedd00** [reply]
I only changed the layout, i tried to write the password as a password for the wifi to check if it written right and it was completely right and i copied it and paste it but it still didnt work too

**Reply by jeremypw** [reply]
The information given here - https://askubuntu.com/questions/514284/how-can-i-change-the-keyboard-layout-on-login-screen - may help.

**Reply by sedd00** [reply]
Thanks
